src/python/pants/init/plugin_resolver.py

```diff
@@ -128,13 +128,16 @@ def resolve(
         env: CompleteEnvironment,
     ) -> WorkingSet:
         """Resolves any configured plugins and adds them to the working_set."""
-        for resolved_plugin_location in self._resolve_plugins(
-            options_bootstrapper, env, self._request
-        ):
-            site.addsitedir(
-                resolved_plugin_location
-            )  # Activate any .pth files plugin wheels may have.
-            self._working_set.add_entry(resolved_plugin_location)
+        from pants import ox
```

Comment:
Does this import need to be lazy? If not, moving to the head of the file would be good. If so, a comment would be good.